src/components/document/FamilyDocument.tsx

```diff
@@ -1,37 +1,38 @@
 import { useRouter } from "next/router";
 import MatchesButton from "@components/buttons/MatchesButton";
-import { formatDate } from "@utils/timedate";
-import { GlobeIcon, PDFIcon } from "@components/svg/Icons";
-import { TDocumentPage } from "@types";
+import { TDocumentContentType, TDocumentPage } from "@types";
 
 type TProps = {
   document: TDocumentPage;
-  date: string;
   matches?: number;
 };
 
-export const FamilyDocument = ({ document, date, matches }: TProps) => {
+export const FamilyDocument = ({ document, matches }: TProps) => {
   const { title, slugs, variant, content_type } = document;
   const router = useRouter();
-  const [year, _day, _month] = formatDate(date);
-  const isMain = variant === "MAIN";
-  const hasMatches = typeof matches !== "undefined";
-  const canView = !!document.cdn_object;
+  const isMain = variant.toLowerCase().includes("main");
+  const hasMatches = typeof matches !== "undefined" && matches > 0;
+  // PDFs need to have a cdn location
+  // HTMLs need a source url / website
+  const canView =
+    document.content_type === "application/pdf" ? !!document.cdn_object : document.content_type === "text/html" ? !!document.source_url : false;
```

Comment:
This is very opaque / difficult to read :grin:

Reply:
😁 I'll refactor it when we get more settled on the doc types